apps/lab2EntryPoints.ts

```diff
@@ -16,12 +16,12 @@ import {PythonlabEntryPoint} from '@cdo/apps/pythonlab/entrypoint';
 import {StandaloneVideoEntryPoint} from '@cdo/apps/standaloneVideo/entrypoint';
 import {Weblab2EntryPoint} from '@cdo/apps/weblab2/entrypoint';
 
-export const lab2EntryPoints: Record<string, Lab2EntryPoint> = {
+export const lab2EntryPoints = {
   aichat: AIChatEntryPoint,
   dance: DanceEntryPoint,
   music: MusicEntryPoint,
   panels: PanelsEntryPoint,
   pythonlab: PythonlabEntryPoint,
   standalone_video: StandaloneVideoEntryPoint,
   weblab2: Weblab2EntryPoint,
-};
+} as const satisfies Record<string, Lab2EntryPoint>;
```

Comment:
We use the keys of this object to automatically compute the `AppName` type (i.e. any valid Lab2 app should have a corresponding entrypoint). However, since this object was typed as `Record<string, Lab2EntryPoint>`, the type of `AppName` was just evaluating to `string`. With this change, the `as const` makes it so the type of the object is interpreted literally, so `AppName` evaluates to `'aichat' | 'dance' | ...`, while the `satisfies` ensures that the values are still `Lab2EntryPoint`s.